inference-engine/tests/functional/plugin/shared/src/subgraph_tests/split_trivial_permute_concat.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include <debug.h>
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "subgraph_tests/split_trivial_permute_concat.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+
+
+namespace LayerTestsDefinitions {
+    std::string SplitTrivialPermuteConcatTest::getTestCaseName(const testing::TestParamInfo<SplitTrivialPermuteConcatTuple>& obj) {
+        InferenceEngine::Precision netPrecision;
+        std::string targetName;
+        std::tie(netPrecision, targetName, std::ignore) = obj.param;
+        std::ostringstream results;
+
+        results << "netPRC=" << netPrecision.name() << "_";
+        results << "targetDevice=" << targetName << "_";
+        return results.str();
+    }
+
+    void SplitTrivialPermuteConcatTest::SetUp() {
+        InferenceEngine::Precision netPrecision;
+        std::map<std::string, std::string> config;
+        std::tie(netPrecision, targetDevice, config) = this->GetParam();
+        configuration.insert(config.begin(), config.end());
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto input = ngraph::builder::makeParams(ngPrc, { {16, 1, 2, 64} });
```

Comment:
Please try to make test more generic to test various combinations of input sizes, split and concat.